src/main/java/org/sonatype/gradle/plugins/scan/common/DependenciesFinder.java

```diff
@@ -119,7 +121,11 @@
 
   @VisibleForTesting
   Module buildModule(Project project) {
-    Module module = new Module().setIdKind("gradle").setPathname(project.getProjectDir());
+    Module module = new Module()
+        .setIdKind("gradle")
+        .setPathname(project.getProjectDir())
+        .setBuilderInfo(Module.BI_CLM_TOOL, "gradle")
+        .setBuilderInfo(Module.BI_CLM_VERSION, PluginVersionUtils.getPluginVersion());
```

Comment:
Why is this needed out of curiosity?

Reply:
> Why is this needed out of curiosity?

Not sure if needed, but the Maven plugin is adding those fields in the `module.xml` files: https://github.com/sonatype/clm-maven-plugin/blob/13783077fe95d411ed641eb53dfb7a6782838b3b/clm-maven-plugin/src/main/java/com/sonatype/insight/maven/IndexMojo.java#L45-L46